examples/rcv1.d

```diff
@@ -122,9 +122,9 @@ class RCV1Reader : DataFileReader!(Obs) {
         auto lab_end = countUntil(buff, "  ");
         if(lab_end == -1)
             return false;
-        auto label = labels[to_long(buff[0..lab_end])];
+        auto label = labels[buff[0..lab_end].to!size_t];
```

Comment:
we should probably do something like `to_long(buff[0..lab_end]).to!size_t` to keep the fast (but unsafe) parsing through [to_long](https://github.com/Netflix/vectorflow/blob/master/src/vectorflow/utils.d#L45)

Reply:
Yes.